### ISSUE TITLE
A space between class name and "[]" breaks the array declaration.

### ISSUE THREAD

**Issue by felixhao28** [atom/language-java#196]
### Description

Putting a space between class name and "[]" breaks the array declaration.

![image](https://user-images.githubusercontent.com/2417391/55850679-f1fb4680-5b87-11e9-8c70-7e9b5dfbf5bf.png)

```java
package com.sell.controller; 
import java.util.*;

public class testbug{

    public static void main(String [] args){
        String str = "";
        String [] arr = str.split("s");
//            ^ space
        for (int i = 0 ; i < arr.
```

Expected:

![image](https://user-images.githubusercontent.com/2417391/55850042-62549880-5b85-11e9-9619-4eec00dde61c.png)

I fixed it by replacing

https://github.com/atom/language-java/blob/27fa88a4fc76649fa6dc4ea049708bb10fa0f1e7/grammars/java.cson#L1432

to 

`(\\s*\\[\\])* # int[][]`

**Comment by sadikovi** [comment]
Thanks, I will try patching it tonight.

**Comment by sadikovi** [comment]
Fixed in #197.
